pyramid_swagger/__about__.py

```diff
@@ -13,7 +13,7 @@
 __summary__ = "Swagger tools for use in pyramid webapps"
 __uri__ = "https://github.com/striglia/pyramid_swagger"
 
-__version__ = "2.8.0"
```

Comment:
Can you seperate the version change and changelog change into follow-up PR? This way, the changes can be treated independently of the version change.

Reply:
done